### PR TITLE
feat(pmml): Support PMML signature

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #
https://github.com/caicloud/ormb/issues/62

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/model/types.go [hunk 1]
```diff
@@ -45,6 +45,10 @@ type Tensor struct {
 	Name  string `json:"name,omitempty" yaml:"name,omitempty"`
 	Size  []int  `json:"size,omitempty" yaml:"size,omitempty"`
 	DType string `json:"dtype,omitempty" yaml:"dtype,omitempty"`
+	// OpType is special for PMML
+	OpType string `json:"optype,omitempty" yaml:"optype,omitempty"`
```
Comment (gaocegege) [inline pkg/model/types.go:49]: Should we define all the string to `*string` since it is optional?

Comment (simon-cj) [reply pkg/model/types.go:49]: Why optional field is must be `*string`? I think whether it is `string` or `*string`,  the `omitempty` is effective.

Comment (gaocegege) [reply pkg/model/types.go:49]: It's more like a syntactic sugar, maybe.

If we use *string, we can check if the value is set using `if opType == nil`.

Just one nit, doesn't matter.

/lgtm

**Pull request merged**